draft-ietf-mls-protocol.md

```diff
@@ -4214,6 +4205,67 @@ Regardless of how the client obtains the tree, the client MUST verify that the
 root hash of the ratchet tree matches the `tree_hash` of the GroupContext before
 using the tree for MLS operations.
 
+## Proposal-List Validation
```

Comment:
```suggestion
## Proposal List Validation
```

Reply:
I would put this above, probably as a subsection of {{proposals}}.  But we can handle that in a section reordering PR; I have added this to #672